parm/config/gefs/config.ufs

```diff
@@ -92,19 +88,13 @@ case "${fv3_res}" in
         export knob_ugwp_tauamp=6.0e-3      # setting for UGWPv1 non-stationary GWD
         export WRITE_GROUP=1
         export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE=1
-        export WRITE_GROUP_GFS=1
-        export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GFS=1
         ;;
     "C96")
         export DELTIM=600
         export layout_x=2
         export layout_y=2
-        export layout_x_gfs=6
-        export layout_y_gfs=8
```

Comment:
These `_gfs` values are different than the gdas (non _gfs) ones. Are these values somewhere else now or perhaps no longer needed (gfs runs fine with 2 and 2)? I have similar concerns for some of the other resolutions.

Reply:
Ah, ok, thanks for clarifying!